package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@istreamplanet/pebble",
-  "version": "2.9.0",
+  "version": "2.9.1",
```

Comment:
Is this manually change?

Reply:
We should be letting the npm commandline bump the version rather than changing it directly.

Reply:
reverted